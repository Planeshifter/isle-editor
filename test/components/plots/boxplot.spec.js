// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import BoxPlot from '@isle-project/components/plots/boxplot';
import heartdisease from './heartdisease.json';


// TESTS //

describe( '<BoxPlot />', function test() {
	it( 'renders a box plot', () => {
		const chart = renderer.create( <BoxPlot
			data={heartdisease}
			variable="Age"
		/> );
		expect( chart ).toMatchSnapshot();
	});
});
