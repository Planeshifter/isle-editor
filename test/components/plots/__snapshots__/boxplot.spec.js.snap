// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoxPlot /> renders a box plot 1`] = `null`;
